File: 75-sort-colors/sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        int countZero=0;
        int countOne=0;
        int countTwo=0;

        int n = nums.length;

        for(int i=0;i<n;i++){
            if(nums[i]==0) countZero++;
            else if(nums[i]==1) countOne++;
            else countTwo++;
        }

        int i=0;
       while(countZero > 0){
        nums[i]=0;
        i++;
        countZero--;
       }
        
       while(countOne > 0){
        nums[i]=1;
        i++;
        countOne--;
       }
        
       while(countTwo > 0){
        nums[i]=2;
        i++;
        countTwo--;
       }
    }   
}

File: 75-sort-colors/sort-colors.js
/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */
var sortColors = function(nums) {

    let n = nums.length;

        let l=m=0,h=n-1;

        while(m<=h){
            if(nums[m]==0){
                nums[m]=nums[l];
                nums[l]=0;
                l++;
                m++;

            }else if(nums[m]==1){
                m++;

            }else{
                nums[m]=nums[h];
                nums[h]=2;
                h--;
            }
        }
    
};